anaInfo.py

```diff
@@ -61,6 +61,15 @@ class MaskReason:
     HighNoise   = 0x08
     HighEffPed  = 0x10
 
+    @staticmethod
+    def list():
```

Comment:
find a different name for this `list` is too close to the actual built in type, if you prefer `listOfTuples()` but this will cause confusion later on. e.g. `list()` vs `list[]` are too similar.